easybuild/easyconfigs/p/PIRATE/PIRATE-1.0.5-foss-2022a.eb

```
@@ -0,0 +1,40 @@
+easyblock = 'Tarball'
+
+name = 'PIRATE'
+version = '1.0.5'
```

this should carry the R versionsuffix too
```suggestion
version = '1.0.5'
versionsuffix = '-R-%(rver)s'
```

---

fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/17275/commits/d00bd2f451a8fc7317acc07a399f62a6ffa4f803